### PR TITLE
Continue migrating #135 (letter 'h')

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2876,12 +2921,17 @@ hopfully->hopefully
...
+horzontal->horizontal
+horzontally->horizontally
 hosited->hoisted
 hospitible->hospitable
 hould->hold, should,
 hounour->honour
 housr->hours, house,
 hove->hove, have, hover, love,
+hovewer->however
+how log->how long
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: this one has legitimate uses

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @larsoner I was trying to think of some. Care to give an example ?

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: "We have to do this because of how log files work"

**Comment by luzpaz** [comment]
Removed offending entry. Ready to merge.

**Pull request merged**